ambry-clustermap/src/test/java/com/github/ambry/clustermap/HelixClusterManagerTest.java

```diff
@@ -898,6 +901,37 @@ public void helixInitiatedIdealStateChangeTest() throws Exception {
         .containsKey(partitionName));
   }
 
+  /**
+   * Tests the isValidPartitionFilter function
+   */
+  @Test
+  public void helixClusterPartitionFilterTest() throws Exception {
+    assumeTrue(!useComposite && !overrideEnabled);
+    String partitionID = "22";
```

Comment:
Let's not use a random partition id here, we only have one resource in the helix cluster for this test, so any partition id should be good for testing. But you don't really know if the partition id 22 exists or not, could you do something like
```
String partitionId = helixClusterManager.getPartitions().iter().next().toString(); 
```
to return a partition id that we know definitely exist.